infra-as-code/bicep/modules/vwanConnectivity/bicepconfig.json

```diff
@@ -0,0 +1,69 @@
+{
+  "analyzers": {
+    "core": {
+      "enabled": true,
+      "verbose": true,
+      "rules": {
+        "adminusername-should-not-be-literal": {
+          "level": "error"
+        },
+        "no-hardcoded-env-urls": {
+          "level": "error",
+          "disallowedhosts": [
```

Comment:
Are these needed as you are not deploying the DNS zones which is why we added these for the hub networking module?

Reply:
you got me, it was copy and paste from hub networking module, lol. fixed